bundles/EcommerceFrameworkBundle/VoucherService/Token/Listing.php

```diff
@@ -20,13 +20,19 @@
 
 /**
  * @method Token[] load()
+ * @method Token current()
  * @method int getTotalCount()
  * @method \Pimcore\Bundle\EcommerceFrameworkBundle\VoucherService\Token\Listing\Dao getDao()
  */
-class Listing extends \Pimcore\Model\Listing\AbstractListing implements \Iterator, AdapterInterface, AdapterAggregateInterface
+class Listing extends \Pimcore\Model\Listing\AbstractListing implements AdapterInterface, AdapterAggregateInterface
 {
     public $tokens;
```

Comment:
I guess we should mark those "replaced" properties in the listing classes as `@deprecated` to that we can remove them in v7.

Reply:
Done